hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -52,6 +52,32 @@
                         scmused : "N/A",
                         remaining : "N/A",
                         nonscmused : "N/A"
+                    },
+                    data : {
+                        pipeline : {
+                            closed : "N/A",
+                            allocated : "N/A",
+                            open : "N/A",
+                            dormant : "N/A"
+                        },
+                        container : {
+                            open : "N/A",
+                            closing : "N/A",
+                            quasi_closed : "N/A",
+                            closed : "N/A",
+                            deleting : "N/A",
+                            deleted : "N/A",
+                            recovering : "N/A",
+                            under_replicated : "N/A",
+                            mis_replicated : "N/A",
+                            over_replicated : "N/A",
+                            missing : "N/A",
+                            unhealthy : "N/A",
+                            empty : "N/A",
+                            open_unhealthy : "N/A",
+                            quasi_closed_stuck : "N/A",
+                            open_without_pipeline : "N/A"
+                        }
```

Comment:
Could you flatten the hierarchy for different types of metrics? In my opinion, pipelines and containers do not "belong" to "nodes" so they should not be nested under it.

I suggest something like:

- datanode: `statistics.nodes`
- pipeline: `statistics.pipelines`
- containers: `statistics.containers`